src/OpenTK.Windowing.Desktop/Win32WindowProc.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using OpenTK.Windowing.GraphicsLibraryFramework;
+
+namespace OpenTK.Windowing.Desktop
+{
+    /// <summary>
+    /// Windows-specific Win32 message-handling.
+    /// </summary>
+    internal unsafe class Win32WindowProc : IDisposable
+    {
+        [DllImport("user32.dll", EntryPoint = "SetWindowLong")]
+        private static extern int SetWindowLong32(IntPtr hWnd, int nIndex, int dwNewLong);
+
+        [DllImport("user32.dll", EntryPoint = "SetWindowLongPtr")]
+        private static extern IntPtr SetWindowLongPtr64(IntPtr hWnd, int nIndex, IntPtr dwNewLong);
```

Comment:
Remove the 64 postfix. Should be `SetWindowLongPtr`.

Reply:
Pending consensus on the `SetWindowLongPtrW` request...